package.json

```diff
@@ -71,6 +71,8 @@
     "test:ui": "vitest run -r packages/ui --passWithNoTests --coverage",
     "test:tools": "vitest run tools --passWithNoTests --coverage",
     "test:watch": "vitest watch",
+    "test-watch:renderer": "npm run build:ui && vitest -c packages/renderer/vite.config.js watch packages/renderer --passWithNoTests",
```

Comment:
not sure why in watch mode we run build:ui first

Reply:
see slack with errors I got, if podman-desktop:ui hasn't been updated in package.json vs what we have in main, we have to build it before running tests, or else it fails.

Reply:
in order to follow convention should be rename test:watch:renderer